README.md

```diff
@@ -4,18 +4,30 @@ The name comes from Marie Jean Antoine Nicolas de Caritat, Marquis of Condorcet,
 French philosopher and mathematician, notably known for championing an election
 method that now named after him.
 
-The goal of this project is to allow organisations of people working remotely to
-cast votes is a secure and transparent way, using a git repository to collect and
+The goal of this project is to allow organizations of people working remotely to
+cast votes in a secure and transparent way, using a git repository to collect and
 authenticate votes.
 
+## Requirements
+
+To be able to cast a vote or create a ballot you will
+need the following tools:
+
+- [Node.js v16+](https://nodejs.org)
+- [git](https://git-scm.com)
+
+If you want to cast a vote with GitHub verified signature
+you will also need to install [PGP](https://www.openpgp.org/).
+Later in the documentation we will check how to install PGP depending
```

Comment:
PGP is a protocol, you can't install it – like you can't "install HTTP", that wouldn't mean anything. They would need a PGP-client, a `gpg` is what Caritat will expect.